ChangeLog.2012

```diff
@@ -1270,14 +1270,14 @@
 	* [r22424] doc, doc/Makefile.am, doc/gnc-fq-dump.1,
 	  doc/gnc-fq-helper.1: Bug #684719 - Man pages for gnc-fq-* perl
 	  scripts first (static) publishing of the pod extracted man pages
-	  until we have some consense.
+	  until we have some consense [??].
```

Comment:
Not sure what's intended

Reply:
"consensus"

Reply:
https://github.com/Gnucash/gnucash/pull/628/commits/e285a9325d9b3c3d14ce12d84d263c4c3306452a